javalib/src/main/scala/scala/scalanative/nio/fs/attribute/NativeUserPrincipalLookupService.scala

```diff
@@ -0,0 +1,108 @@
+package scala.scalanative.nio.fs.attribute
+
+import java.nio.file.attribute._
+
+import scalanative.unsigned._
+import scalanative.unsafe._
+import scalanative.libc._
+import scalanative.posix.{errno => e, grp, pwd, unistd, time, utime}, e._
+import scalanative.posix.sys.stat
+import scala.scalanative.nio.fs.UnixException
+
+final case class NativeUserPrincipal(uid: stat.uid_t)(name: Option[String]) extends UserPrincipal {
```

Comment:
This class and the one immediately below should have  `private` between the class name and the opening
parenthesis for the arguments.  As they are now, the arguments are visible to the public, which does not
match the Java 8 description.

Reply:
Fixed.